src/main/frontend/handler.cljs

```diff
@@ -199,6 +199,7 @@
     (render)
     (i18n/start)
     (instrument/init)
+    (state/set-online! js/navigator.onLine)
```

Comment:
set network offline first, then open app(or refresh), :network/online? in state won't be false, we need to set it on start